src/navigators/createBottomTabNavigator.js

```diff
@@ -81,6 +83,7 @@ class TabNavigationView extends React.PureComponent<Props, State> {
     return (
       <TabBarComponent
         {...tabBarOptions}
+        onResize={tabBarHeight => this.setState({ tabBarHeight })}
```

Comment:
```suggestion
        onResize={isWeb && (tabBarHeight) => this.setState({ tabBarHeight })}
```

Reply:
this suggestion may not actually be properly formatted 😅 but basically we should just pass in null for this except on web